src/App.js

```diff
@@ -1,8 +1,10 @@
 import './App.css';
-import React, {useRef} from 'react';
+import React, {useEffect, useRef} from 'react';
 import {FooterNav, Navbar, Navpills, Sidebar} from './comps/index'
-import {Home, Playlist, Liked, Watchlater, History, Signup, Login} from './pages/index';
+import {Home, Playlist, Liked, Watchlater, History, Signup, Login, PageNotFound} from './pages/index';
```

Comment:
`/index` is not required as it will check the index files for exports if you do not mention any file